packages/models-library/src/models_library/invitations.py

```diff
@@ -39,7 +39,7 @@ class InvitationInputs(BaseModel):
     @classmethod
     def trim_long_issuers_to_max_length(cls, v):
         if v and isinstance(v, str):
-            return v[: _MAX_LEN - 1]
```

Comment:
@pcrespov 
any reason why this is 39 char long and not 40 ?
I have removed the -1 since it made no sense to me. If it needs to stay -1 I refactor the code.

Reply:
As we discussed, because 39 < 40 :-) and I thought `max_length` was a strict condition but it seems to be <=40